java/test/jmri/util/prefs/JmriPreferencesProviderTest.java

```diff
@@ -105,7 +105,7 @@ public void testIsFirstUse(TestInfo info, @TempDir File folder) throws IOExcepti
     /**
      * Test of findCNBForPackage method, of class JmriPreferencesProvider.
      */
-    @Test
+    //@Test - fails when run separately die to Package.getPackage returning null
     public void testFindCNBForPackage() {
         assertEquals("jmri-server-json", JmriPreferencesProvider.findCNBForPackage(Package.getPackage("jmri.server.json")));
     }
```

Comment:
This code works and it has the benefit of not using the deprecated method `Package.getPackage()`.
```
    /**
     * Test of findCNBForPackage method, of class JmriPreferencesProvider.
     */
    @Test
    public void testFindCNBForPackage() {
        ClassLoader cl = getClass().getClassLoader();
        assertEquals("jmri-util", JmriPreferencesProvider.findCNBForPackage(cl.getDefinedPackage("jmri.util")));
        assertEquals("jmri-jmrit-logixng", JmriPreferencesProvider.findCNBForPackage(cl.getDefinedPackage("jmri.jmrit.logixng")));
    }
```

It doesn't matter which package you look for, as long as that package is already loaded by a class loader.

Reply:
Thanks!  I think the problem was that when running one-at-a-time, the classloader hadn't been asked to load jmri.server.json.

I'll change to your suggestion, Thanks again.